plugin/vimteractive.vim

```diff
@@ -29,13 +29,8 @@ if !has_key(g:, 'vimteractive_commands')
 	let g:vimteractive_commands = { }
 endif
 
-
-let g:vimteractive_commands.ipython = 'ipython --matplotlib --no-autoindent'
-let g:vimteractive_commands.ipython2 = 'ipython2 --matplotlib --no-autoindent'
-let g:vimteractive_commands.ipython3 = 'ipython3 --matplotlib --no-autoindent'
+let g:vimteractive_commands.ipython = 'ipython --matplotlib --no-autoindent --logfile="-o /tmp/vimteractive_ipython-UID"'
```

Comment:
Writing to `/tmp` (here and for the new REPL) should probably respect `TMPDIR` (though I’m not sure how to do that in a cross-platform fashion). I’m also not sure if ipython would clean up this file after exiting? If not, this would create an unbounded amount of files in `/tmp`, which could be problematic in the long term if it’s not on tmpfs (and eat up RAM if it is).

Reply:
OK, this is now addressed in 64cfad0. We now make use of the vim `tempfile()` function which creates a temporary directory which is closed when the buffer finishes. Vim's automatic filenaming means that the UIDs are no longer needed.

This is considerably neater, and I learnt something about how vim temporary files work, so thank you very much for this contribution!